packages/manager/modules/pci/src/projects/onboarding/onboarding.component.js

```diff
@@ -4,6 +4,15 @@ import template from './onboarding.html';
 export default {
   bindings: {
     isTrustedZone: '<',
+    goToCreateNewProject: '<',
+    goToCreateDiscoveryProject: '<',
+    cart: '<',
+    isValidHdsSupportLevel: '<',
+    hds: '<',
+    model: '<',
+    summary: '<',
+    getSummary: '<',
```

Comment:
```suggestion
    getSummary: '<',
```
`summary` seems to be useless

Reply:
The `summary` attribute need to be declared here to load the user contract data on the first page loading.
Removing it will break the page.